etna/transforms/timestamp/holiday.py

```diff
@@ -23,12 +24,13 @@ def __init__(self, iso_code: str = "RUS", out_column: Optional[str] = None):
         out_column:
             name of added column. Use ``self.__repr__()`` if not given.
         """
+        super().__init__(required_features="all")
```

Comment:
This transform actually does not require any features, so there should be ["target"] just as placeholder